drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/PhantomJSDriverFactory.java

```diff
@@ -35,17 +33,14 @@
  * PhantomJSDriver.
  *
  * @author <a href="kpiwko@redhat.com">Karel Piwko</a>
- *
+ * @author <a href="mjobanek@redhat.com">Matous Jobanek</a>
```

Comment:
Maybe it would be best to remove them as a content of one separate commit (including formatting changes). Jira issue would be great.